test/Dialect/FIRRTL/grand-central-signal-mappings.mlir

```diff
@@ -26,9 +26,9 @@ firrtl.circuit "SubCircuit" attributes {
   // CHECK:         firrtl.force [[T2]], %clock_sink
   // CHECK:       }
   // CHECK-LABEL: firrtl.module @Foo
-  firrtl.module @Foo() attributes {annotations = [{class = "sifive.enterprise.grandcentral.SignalDriverAnnotation", id = 0 : i64}]} {
-    %clock_source = firrtl.wire {annotations = [{class = "sifive.enterprise.grandcentral.SignalDriverAnnotation", dir = "source", id = 0 : i64, peer = "~Main|Main>clock", side = "local", targetId = 0 : i64}]} : !firrtl.clock
-    %clock_sink = firrtl.wire {annotations = [{class = "sifive.enterprise.grandcentral.SignalDriverAnnotation", dir = "sink", id = 0 : i64, peer = "~Main|Main>clock", side = "local", targetId = 1 : i64}]} : !firrtl.clock
+  firrtl.module @Foo() attributes {annotations = [{class = "sifive.enterprise.grandcentral.SignalDriverAnnotation.module", id = 0 : i64}]} {
+    %clock_source = firrtl.wire {annotations = [{class = "sifive.enterprise.grandcentral.SignalDriverAnnotation.target", dir = "source", id = 0 : i64, peer = "~Main|Main>clock", side = "local", targetId = 0 : i64}]} : !firrtl.clock
+    %clock_sink = firrtl.wire {annotations = [{class = "sifive.enterprise.grandcentral.SignalDriverAnnotation.target", dir = "sink", id = 0 : i64, peer = "~Main|Main>clock", side = "local", targetId = 1 : i64}]} : !firrtl.clock
```

Comment:
While it may be necessary, I like having these annotations differentiated (`.module` vs `.target` I mean)!

Reply:
It came out of some necessity to do different application of the annotations in `LowerAnnotations` when recursively expanding one annotation into multiple. To feed these to different appliers, they needed to have different classes.

I actually think it's pretty clear and has been rolled out in GCT Views and Taps as well.